app/views/decks/show.html.erb

```diff
@@ -1,4 +1,10 @@
 <table class="table is-bordered">
+  <div class="buttons">
+    <%= link_to 'Edit Deck', "/players/#{@player.id}/decks/#{@deck.id}/edit", class: 'button is-warning' %>
+    <%= link_to 'Delete Deck', "/players/#{@player.id}/decks/#{@deck.id}",
```

Comment:
Since you're already on the deck's page it's redundant information to have `Deck` in the button's labels. I would have an `Edit` and a `Delete` button.